lib/active_record/connection_adapters/oracle_enhanced/schema_statements.rb

```diff
@@ -118,11 +118,23 @@ def dump_schema_information #:nodoc:
         end
 
         def insert_versions_sql(versions) # :nodoc:
-          sm_table = ActiveRecord::Migrator.schema_migrations_table_name
+          sm_table = quote_table_name(ActiveRecord::Migrator.schema_migrations_table_name)
 
-          versions.map { |version|
-            "INSERT INTO #{sm_table} (version) VALUES ('#{version}')"
-          }.join "\n\n/\n\n"
+          if self.supports_multi_insert?
```

Comment:
Would you tell me if there is a reason it uses `self`. If not would you remove it.